src/Services/ShipmentNotificationService.php

```diff
@@ -331,20 +332,19 @@ public function prepareRequestDto(Order $order): RequestDTO
         return null;
       }
 
-      $plentymarketsShippingInformation = $this->shippingInformationRepositoryContract->getShippingInformationByOrderId($orderId);
-      if (!isset($plentymarketsShippingInformation) || empty($plentymarketsShippingInformation)) {
-        $this->loggerContract
-          ->error(
-            TranslationHelper::getLoggerKey(self::LOG_KEY_PM_MISSING_SHIPPING_INFO), [
-              'additionalInfo' => [
-                'orderId' => $orderId,
-                'poNumber' => $poNumber
-              ],
-              'method' => __METHOD__,
-              'referenceType' => '$orderId',
-              'referenceValue' => $orderId
-            ]
-          );
+      $plentymarketsShippingInformation = json_encode ($this->shippingInformationRepositoryContract->getShippingInformationByOrderId($orderId));
+      if (!isset($plentymarketsShippingInformation) || empty($plentymarketsShippingInformation) || empty($plentymarketsShippingInformation->shippingServiceProvider)) {
```

Comment:
how is `$plentymarketsShippingInformation->shippingServiceProvider` not throwing errors if you made it a `string` on the line above this one?